src/assets/_project/_blocks/components/accordion/qg-accordion.js

```diff
@@ -162,13 +162,13 @@ export class QgAccordion {
    * @return {undefined}
    **/
   legacyAccordion(){
-    let self = this;
-    let accItem = $('.qg-accordion:not(.qg-accordion-v2)').find('article');
+    const self = this;
```

Comment:
don't think these can be const. Is this code stood up multiple times for each accordion?